patrick/instagram/src/CommentSection.js

```diff
@@ -0,0 +1,100 @@
+// Comment Section Component
+// should render an input tag
+// should call componentDidMount
+// should store comments in state
+// should add comments to state upon mounting
+
+import React, { Component } from 'react';
+
+
+class CommentSection extends Component {
+  constructor(props) {
+    super(props);
+    // this.state = {
```

Comment:
Comments inside files are fine if they're descriptors of what the code is doing, but commented out code should be beleted.